### PR TITLE
e2e- Add video and trace recording to global setup for debugging assistance

### PR DESCRIPTION
**Pull request by bsessions85**
## Description

This PR adds video recording and trace logging for the global setup script ( main tests already have this). It also causes the tests to fail fast if the login doesn't work so we're not wasting resources if we weren't able to log in.

I was also having an issue with running tests in cli and vs code and getting different results due to path. Made some changes to fix that.

## Checklist

Please make sure the items below have been covered before requesting a review:

- [x] This change works and has been tested locally (or has an appropriate fallback).
- [ ] This change works and has been tested on a Go sandbox.
- [ ] This change has relevant unit tests (if applicable).
- [ ] This change has relevant documentation additions / updates (if applicable).
- [ ] I've created a changelog description that aligns with the provided examples.

## Steps to Test
Ensure tests pass in CI.

Run locally using `npm run setup-e2e-env` followed by `npm run test-e2e`
Make sure the tests pass and that there is a directory created with video and trace for the setup.

### CODE REVIEW

File: __tests__/e2e/lib/global-setup.ts [hunk 1]
```diff
@@ -12,40 +12,57 @@ import { SettingsWritingPage } from './pages/settings-writing-page';
...
     // Save signed-in state
     await page.context().storageState( { path: storageState as string } );
 
-    await browser.close();
+    await context.tracing.stop( { path: artifactsDir + 'trace.zip' } );
+    await context.close();
+
+    if ( ! success ) {
+        throw new Error( 'Setup unsuccessful - Tests will not run' )
+    }
```
Comment (sjinks) [inline __tests__/e2e/lib/global-setup.ts:65]: @bsessions85 should we clean up the video and trace on success?

Comment (bsessions85) [reply __tests__/e2e/lib/global-setup.ts:65]: @sjinks yeah, we probably should. I'll get that in there. Thanks!

Comment (bsessions85) [reply __tests__/e2e/lib/global-setup.ts:65]: @sjinks  Got that added in. Thanks!

**Comment by bsessions85** [comment]
Had some linting issues on file cleanup. Will look in to those on Monday. They are security related so need to make sure I do it properly.

**Comment by bsessions85** [comment]
Closing this one, The branch got all messed up and shouldn't be this complicated

**Pull request closed without merging**